nuitka/tools/specialize/templates_python/AttributeNodeFixed.py.j2

```diff
@@ -412,3 +425,97 @@ class ExpressionAttributeLookupBytes{{attribute_name_class}}(SideEffectsFromChil
 attribute_typed_classes.add(ExpressionAttributeLookupBytes{{attribute_name_class}})
 
 {% endif %}
+
+{% if "tshape_list" in shape_names %}
+
+{% if list_operation_version_presence %}
+from nuitka.specs.BuiltinListOperationSpecs import list_{{attribute_name}}_spec
+{% if python3_operation_name %}
+from .ListNodes import ExpressionListOperation{{attribute_name_class}}
+from .ListNodes import ExpressionListOperation{{python3_operation_name.title()}}
+{% else %}
+{% if list_operation_variation %}
+{% for count in reversed(list_operation_variation[1:]) %}
+from .ListNodes import ExpressionListOperation{{attribute_name_class}}{{count+1}}
+{% endfor %}
+from .ListNodes import ExpressionListOperation{{attribute_name_class}}{{list_operation_variation[0]+1}}
+{% else %}
+from .ListNodes import ExpressionListOperation{{attribute_name_class}}
+{% endif %}
+{% endif %}
+{% endif %}
+
+class ExpressionAttributeLookupList{{attribute_name_class}}(SideEffectsFromChildrenMixin, ExpressionAttributeLookupFixed{{attribute_name_class}}):
+    """Attribute {{attribute_name_class}} lookup on a list.
+
+    Typically code like: some_list.{{attribute_name}}
+    """
+
+    kind = "EXPRESSION_ATTRIBUTE_LOOKUP_LIST_{{attribute_name_class.upper()}}"
+    attribute_name = "{{attribute_name}}"
+
+    def computeExpression(self, trace_collection):
+        return self, None, None
+
+{% if list_operation_version_presence %}
+    @staticmethod
+    def _computeExpressionCall(call_node, list_arg, trace_collection):
+        def wrapExpressionListOperation{{attribute_name_class}}({{formatArgs(list_operation_args, starting=True, finishing=False)}} source_ref):
+{% if python3_operation_name %}
+            if str is list:
```

Comment:
I do not think this test makes sense. It seems like a bad copy-paste/bad replace to me. AFAIK, the original test (str is bytes) is just a way to test for Python3 vs Python2 but "str is list" will always be False, right?

Reply:
Yes, gonna change that.
I replaced bytes with list, and didn't notice that.